### PR TITLE
[17.0][MIG] document_page_quality_manual: Migration to 17.0

### PR DESCRIPTION
**Pull request by jaydeep32**

### CODE REVIEW

**Comment by mymage** [comment]
@jaydeep32 please re-push to check test

File: document_page_quality_manual/pyproject.toml
Comment (mymage) [inline document_page_quality_manual/pyproject.toml:1]: This file must be deleted

**Review by mymage — CHANGES_REQUESTED** [review]

**Review by mymage — APPROVED** [review]
LGTM

**Comment by mymage** [comment]
Hi @jaydeep32, the repo was fixed (see #585).
Then I think you have to re-push this after a rebase. Thanks!

**Comment by max3903** [comment]
Superseeded by #595

**Pull request closed without merging**